index.html

```diff
@@ -10,87 +11,103 @@
+			<h2>Recent Projects</h2>
+				<div class="project">
+					<div class="text-block villas"><h3>THE VILLAS</h3></div>
+					<p>The Villas bring to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p><br> 
+					<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.</p>
+				</div>
+				<div class="project">
+					<div class="text-block2 outskirts"><h3>OUTSKIRTS</h3></div>
+					<p>The Outskirts are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p><br>
+					<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.</p>
+				</div>
+				<div class="project bottom">
+					<div class="text-block city-block"><h3>THE BLOCKS</h3></div>
+					<p>The Blocks are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution.</p><br> 
+					<p>Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.</p>
+				</div>
+		</section>
+	</div>
+	<!-- FOOTER CONTENT-->
+	<footer>
+		<div class="container">
+			<section class="footer-email">
+				<h4>Interested in starting a project?<br>Let’s talk: </h4>
+				<input type="text" id="email" placeholder="Enter email">
+				<p>We'll never share your email with anyone else.</p>
+			</section>
+			<section class="location-ny">
+				<h4>New York</h4>
+				<address>123 Lane<br>Suite 100<br>Albany, NY 12345</address>
+				<p>202 555 0144</p>
+			</section>
+			<section class="location-fl">
+				<h4>Florida</h4>
+				<address>Ocean Drive<br>Suite 201<br>Orlando, FL 22345</address>
+				<p>502 555 0144</p>
+			</section>
+			<section class="location-ca">
+				<h4>California</h4>
+				<address>Mountain Street<br>Suite 105<br>San Diego, CA 22345</address>
+				<p>702 555 0144</p>
+			</section>
+			
+		</div>
+		<span class="copyright">
+			<p>Copyright © 2018 Smith and Jones</p>
+		</span>
+	</footer>
 	<script src="js/index.js"></script>
 </body>
 </html>
```

Comment:
Perhaps consider using a different spacing for html to clean up the code to make it easier for you. I use 2 for html and line up the tags with each other for cleanliness. Just some styling tips.